js/src/forum/states/DiscussionListState.js

```diff
@@ -88,6 +88,9 @@ export default class DiscussionListState {
 
     return this.loadResults().then(
       (results) => {
+        // Just in case we had 2 requests going at the same time, let's clear our cached list to ensure that
+        // we only take into account the latest set of data.
+        this.clear();
```

Comment:
Clearing without the m.redraw() here would be better. I tested locally that this leads to an additional redraw in mithril v0.2

Reply:
I'm fine changing it back to directly setting this.discussions without the `clear` method. @clarkwinkelmann how would you feel about reverting that change?

Reply:
Is my input still needed on this? Can you clarify? Revert something from which commit/PR?

The force pushes seem to have removed every context possible. I seem to remember the original PR was more than a one-line change.